cordova-lib/src/cordova/plugin.js

```diff
@@ -305,6 +289,50 @@ module.exports = function plugin(command, targets, opts) {
     });
 };
 
+function determinePluginTarget(projectRoot, cfg, target, fetchOptions) {
+    var parts = target.split('@');
+    var id = parts[0];
+    var version = parts[1];
+
+    // If no version is specified, retrieve the version (or source) from config.xml
+    if (!version && !cordova_util.isUrl(id) && !cordova_util.isDirectory(id)) {
```

Comment:
Right, I just refactored this code, but I should have caught that. Changing now.